x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/SpanFlyout/index.tsx

```diff
@@ -55,8 +55,8 @@ export function SpanFlyout({
 
   const stackframes = span.span.stacktrace;
   const codeLanguage = idx(parentTransaction, _ => _.service.language.name);
-  const dbContext = idx(span, _ => _.context.db);
-  const httpContext = idx(span, _ => _.context.http);
+  const dbContext = idx(span, _ => _.span.db);
+  const httpContext = idx(span, _ => _.span.http);
```

Comment:
These used to be suffixed with "Context" because they were on the context. This should be changed to simply "db" and "http".